modules/cli/src/main/scala/scala/cli/commands/Repl.scala

```diff
@@ -182,44 +211,58 @@ object Repl extends ScalaCommand[ReplOptions] {
     directories: scala.build.Directories,
     logger: Logger,
     allowExit: Boolean,
-    dryRun: Boolean
+    dryRun: Boolean,
+    buildOpt: Option[Build.Successful]
   ): Either[BuildException, Unit] = either {
 
+    val setupPython = options.notForBloopOptions.python.getOrElse(false)
+    val python      = options.notForBloopOptions.python.getOrElse(false)
```

Comment:
`python` and `setupPython` are the same, no?

Reply:
Should be fixed.